internal/service/contentprovider/moduleconfig_test.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
```

Comment:
We need to use yaml library which used by KLM to keep consistency, the one used is `sigs.k8s.io/yaml`, and it's designed for handle k8s resource yaml. I see currently this `gopkg.in/yaml` is widely used by modulectl, we should replace them all,

Reply:
Yeah we have got it all over the place. Let's continue to use it here and I will create a PR after merging this one that is replacing it in the whole project.

Reply:
Created this ticket https://github.com/kyma-project/modulectl/issues/79

I will try to get it in at the side before my vac if time allows.